patches/tModLoader/Terraria/ModLoader/UI/UIModSourceItem.cs

```diff
@@ -40,7 +40,15 @@ internal class UIModSourceItem : UIPanel
 			Width.Percent = 1f;
 			SetPadding(6f);
 
-			string addendum = Path.GetFileName(mod).Contains(" ") ? $"  [c/FF0000:{Language.GetTextValue("tModLoader.MSModSourcesCantHaveSpaces")}]" : "";
+			bool isLoaded = ModLoader.GetMod(mod) != null;
+
+			string addendum = "";
+			if (Path.GetFileName(mod).Contains(" ")) {
+				addendum = $"  [c/FF0000:{Language.GetTextValue("tModLoader.MSModSourcesCantHaveSpaces")}]";
+			}
+			else if (builtMod != null && !isLoaded) {
+				addendum = $"  [c/FF0000:{Language.GetTextValue("tModLoader.ModReloadRequired")}]";
```

Comment:
If you're reviewing my PR and aren't sure what I meant by reusing the "Reload Required" translation, here it is. But I think it should be fine.